docs/content/_index.md

```diff
@@ -43,4 +43,4 @@ sudo mv ./osm /usr/local/bin/osm
 $ osm install
 ```
 
-![OSM Install Demo](https://github.com/openservicemesh/osm/raw/main/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
+![OSM Install Demo](https://github.com/openservicemesh/osm/raw/release-v0.8/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
```

Comment:
In a subsequent change, this link should be made navigable within docs.openservicemesh.io. We agreed to use the docs website as the primary means to make docs available to users, and when viewed on the doc, it will be a better experience keeping it on the website.

Relative paths must be used for this.

Reply:
Yes! Thank you - I will do this next!